initialization/initialization_test.go

```diff
@@ -5,50 +5,60 @@ import (
 	"flag"
 	"fmt"
 	"io"
-	"io/ioutil"
-	"os"
+	"log"
 	"sync"
 	"testing"
 	"time"
 
 	"github.com/spacemeshos/post/config"
 	"github.com/spacemeshos/post/persistence"
 	"github.com/spacemeshos/post/shared"
-	smlog "github.com/spacemeshos/smutil/log"
 	"github.com/stretchr/testify/require"
 )
 
 var (
-	id   = make([]byte, 32)
-	cfg  config.Config
-	opts config.InitOpts
+	id = make([]byte, 32)
```

Comment:
I was on the fence of just inlining it. I will create an `id` variable in every test that needs one. I'm opposed to use a function here, since I see no benefit of

```golang
id := getId()
```

over

```golang
id := make([]byte, 32)
```